Makefile

```diff
@@ -149,6 +149,11 @@ else ifeq ($(CXX), c++)
     endif
 endif
 
+ifeq ($(COVERAGE), 1)
```

Comment:
This is file is generated via `dmake` so changes should be applied via the tool otherwise you might lose them.